ozw-network-visualization-card.js

```diff
@@ -141,11 +142,15 @@ class OZWNetworkVisualizationCard extends HTMLElement {
   _buildLabel(device) {
     var regDevice = this.device_registry[device.node_id];
 
-    var res = regDevice ? "<b>" + regDevice.name + "</b>\n" : "";
+    //Add user's device name for display
```

Comment:
The code is self explanatory, so we can remove this
```suggestion
    //Add user's device name for display
```

Reply:
I am an automotive software QA manager, that answer is never acceptable ;)

Reply:
Well, I'm an automotive software developer :-P
I always say that if code needs comments, it should be rewritten for readability :-D (except for complex cases or performance optimizations that really can't be changed). But we can leave the comment here if you prefer :P

Reply:
Great Thanks. FWIW, comments in our systems are to document the intent, when the code is reviewed, the reviewer can then confirm the intent was implemented.